app/provider/app.go

```diff
@@ -581,7 +580,6 @@ func New(
 					paramsclient.ProposalHandler,
 					ibcproviderclient.ConsumerAdditionProposalHandler,
 					ibcproviderclient.ConsumerRemovalProposalHandler,
-					ibcproviderclient.ConsumerModificationProposalHandler,
```

Comment:
Those will be eventually removed as part of this [work](https://github.com/cosmos/interchain-security/pull/2130).